### DISCUSSION TITLE
KDDockWidgets draw a strang rectangle.

### DISCUSSION THREAD

**Discussion by mrviit** [KDAB/KDDockWidgets#504] [Q&A]
I'm learning this library. So I forked here for updating the example code (https://github.com/mrviit/KDDockWidgets). I also update the code changes for this issue (https://github.com/KDAB/KDDockWidgets/compare/2.1...mrviit:KDDockWidgets:2.1). The result app draw a strange rectangle that I want to change/disable as below image:
![image](https://github.com/KDAB/KDDockWidgets/assets/5145554/73faff15-35ba-4709-ae14-805a014d97a9)
I'm trying to debug to change/disable this rectangle but no hope. Can you help me?

**Comment by iamsergio** [comment]
Hi, try using https://github.com/KDAB/GammaRay and see which QWidget is doing this
